services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+import settings
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+
+    with transaction.atomic():
+        get_user_model().objects.create_user(
+            username=username,
+            password=password,
+            email=email or "",
+            first_name=first_name or "",
+            last_name=last_name or ""

```

Comment:
strange approach. and when some information was not passed it should be None not ""

Reply:
It doesn't work with None. I use "" because than it will be blank. And these fields cannot be Null. Maybe it is ok to use "" here?

Reply:
You are right about Null. You could use if statements here but seems that your solution is also good